components/HowItWorks/HowItWorks.tsx

```diff
@@ -0,0 +1,203 @@
+import { useRef, useState } from "react";
+import styled from "styled-components";
+import { Wrapper as BaseWrapper, Title as BaseTitle } from "components/Widgets";
+import Illustration from "public/assets/illustration.svg";
+import { useIntersectionObserver, useScrollPosition } from "hooks";
+const HowItWorks = () => {
+  const ref = useRef<HTMLDivElement | null>(null);
+  const refTwo = useRef<HTMLDivElement | null>(null);
+  const entryOne = useIntersectionObserver(ref, {});
+  const entryTwo = useIntersectionObserver(ref, {});
+  const isOneVisible = !!entryOne?.isIntersecting;
+
+  const [entryScrollY, setEntryScrollY] = useState(0);
+  const [topRedHeight, setRedTopHeight] = useState(0);
+  if (entryOne) console.log(entryOne);
+  // WIP on animation here.
```

Comment:
lmk when you want help with this — we can have a meeting or we can chat async, whatever you prefer

Reply:
sounds good, I'll probably work on this more today and see how far I can get in a separate PR. I just had some capacity to start on it.